can't run docker-compose start web, docker-command.bash is crashing

<!--
Hello Gitcoiner!

Please use the template below for bugs found within Gitcoin.
If it is general support you need, reach out to us at
gitcoin.co/slack

Provide a general summary of the issue in the title above and use relevant fields below to define the problem.

-->

**Describe the bug**
 Docker container web is crashing and restarting indefinitely,

**To Reproduce**
Clone the repository [gitcoinco/web](https://github.com/gitcoinco/web)
In the master branch,
Execute `docker-compose up -d --build`
After build, 
doing an `docker-compose logs -f web` prints the following error
```
WARNING: Some services (db, ipfs, redis, testrpc, web) use the 'deploy' key, which will be ignored. Compose does not support 'deploy' configuration - use `docker stack deploy` to deploy to a swarm.
Attaching to gitcoinweb_web_1
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
web_1      | /bin/docker-command.bash: line 17: cd: app: No such file or directory
```


**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

- OS: Windows 10

**Additional context**
I am following this [guide](https://docs.gitcoin.co/mk_setup/)

> Yeah, `docker-compose up -d --build` and `docker-compose logs -f web` just are equal to `docker-compose up --build`, maybe you have a wrong directory location, you should be at `web`

I am running docker-compose from the web directory,

### I am running from root directory, of the following directory structure

```
├───.circleci
├───.github
│   └───ISSUE_TEMPLATE
├───app # << app directory is here
│   ├───app
│   │   ├───management
│   │   │   └───commands
│   │   ├───templates
│   │   │   └───shared
│   │   └───tests
│   ├───assets
│   │
├───bin  # there is no app directory inside bin
│       deploy-gh-pages.bash
│       docker-command.bash   # << bash file is here
 . . . . . . 
 . . . 
 . 
└───scripts
    └───debug
```
### If I look at the docker-command.bash
```
#!/usr/local/bin/dumb-init /bin/bash

# Load the .env file into the environment.
if [ "$ENV" == 'staging' ]; then
    # shellcheck disable=SC2046
    export $(grep -v '^#' app/app/.env | xargs)
fi

# Settings
# Web
GC_WEB_WORKER=${GC_WEB_WORKER_TYPE:-runserver_plus}

# General / Overrides
FORCE_PROVISION=${FORCE_PROVISION:-'off'}
FORCE_GET_PRICES=${FORCE_GET_PRICES:-'off'}
```

> `cd app || exit 1`

this where the bash is exiting, its not able to `cd` to `app` directory.
the bash script re
```
# Check whether or not the .env file exists. If not, create it.
if [ ! -f app/.env ]; then
    cp app/local.env app/.env
fi

# Enable Python dependency installation on container start/restart.
if  [ ! -z "${INSTALL_REQS}" ]; then
    echo "Installing Python packages..."
    pip3 install -r ../requirements/test.txt
    echo "Python package installation completed!"
fi

GC_WEB_OPTS="${GC_WEB_WORKER} ${GC_WEB_INTERFACE:-0.0.0.0}:${GC_WEB_PORT:-8000}"

if [ "$VSCODE_DEBUGGER_ENABLED" = "on" ]; then
    pip install ptvsd
    GC_WEB_OPTS="${GC_WEB_OPTS} --nothreading"
    echo "VSCode remote debugger enabled! This has disabled threading!"
fi

if [ "$GC_WEB_WORKER" = "runserver_plus" ]; then
    GC_WEB_OPTS="${GC_WEB_OPTS} --extra-file /code/app/app/.env --nopin --verbosity 0"
fi

# Provision the Django test environment.
if [ ! -f /provisioned ] || [ "$FORCE_PROVISION" = "on" ]; then
    echo "First run - Provisioning the local development environment..."
    if [ "$DISABLE_INITIAL_CACHETABLE" != "on" ]; then
        python manage.py createcachetable
    fi

    if [ "$DISABLE_INITIAL_COLLECTSTATIC" != "on" ]; then
        python manage.py collectstatic --noinput -i other &
    fi

    if [ "$DISABLE_INITIAL_MIGRATE" != "on" ]; then
        python manage.py migrate
    fi

    if [ "$DISABLE_INITIAL_LOADDATA" != "on" ]; then
        python manage.py loaddata initial
    fi
    date >> /provisioned
    echo "Provisioning completed!"
fi

if [ "$FORCE_GET_PRICES" = "on" ]; then
    python manage.py get_prices
fi

if [ "$KUDOS_LOCAL_SYNC" = "on" ]; then
    bash /code/scripts/sync_kudos_listener_local.bash &
    bash /code/scripts/sync_kudos_local.bash &
fi

exec python manage.py $GC_WEB_OPTS

```

--- a/docs/RUNNING_LOCALLY_DOCKER.md
+++ b/docs/RUNNING_LOCALLY_DOCKER.md
@@ -6,6 +6,59 @@ cd web
 cp app/app/local.env app/app/.env
 ```
 
+## Special instructions for Windows WSL contributors
+*If you are using Windows 10 Professional or Enterprise*
+Download Docker Desktop for Windows [here](https://hub.docker.com/editions/community/docker-ce-desktop-windows).
+
+*If you are NOT using Windows 10 Professional or Enterprise*
+Docker Desktop for Windows is not available to your OS. Follow the steps below to install and configure Docker Toolbox:
+1. Follow the installation instructions on the [manual for installing Docker Toolbox on Windows](https://docs.docker.com/toolbox/toolbox_install_windows/).
+2. WSL by default mounts your C: drive on `/mnt/c`, but Docker Toolkit instead expects it to be mounted on `/c/`. To instruct WSL to mount it in the correct location, create a config file in `/etc/wsl.conf` using WSL and enter the following:
+```
+[automount]
+root = /
+options = "metadata"
+```
+3. Lastly, ensure that you are sharing the folders of your project directory to your VirtualBox VM, i.e. if your Gitcoin repository is located in `C:/Projects/web`, you will have to go to the VirtualBox UI, click on `Settings > Shared Folders`, and ensure that there is an entry with a name of `c/Projects` and a path of `C:\Projects`.
+
+Once Docker is installed (either via Docker Desktop for Windows or Docker Toolkit), install the Docker packages on WSL as you normally would for Ubuntu:
+
+```shell
+# Update the apt package list.
+$ sudo apt-get update -y
+
+# Install Docker's package dependencies.
+$ sudo apt-get install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    software-properties-common
+
+# Download and add Docker's official public PGP key.
+$ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+# Verify the fingerprint.
+$ sudo apt-key fingerprint 0EBFCD88
+
+# Add the `stable` channel's Docker upstream repository.
+#
+# If you want to live on the edge, you can change "stable" below to "test" or
+# "nightly". I highly recommend sticking with stable!
+$ sudo add-apt-repository \
+   "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+   $(lsb_release -cs) \
+   stable"
+
+# Update the apt package list (for the new apt repo).
+$ sudo apt-get update -y
+
+# Install the latest version of Docker CE.
+$ sudo apt-get install -y docker-ce
+
+# Allow your user to access the Docker CLI without needing root access.
+$ sudo usermod -aG docker $USER
+```
+
 ## Startup server
 
 *Check that you have Docker and Docker-compose properly installed*

